src/navigation/waypoint.py

```diff
@@ -90,3 +90,8 @@ def evaluate(self, ud) -> str:
             pass
 
         return 'waypoint_traverse'
+
+
+class WaypointState(BaseWaypointState):
```

Comment:
thought we dropped this? @qhdwight @ankithu

Reply:
I did, not sure why this is here

Reply:
merge gone wrong it seems